src/interface.jl

```diff
@@ -559,6 +713,17 @@ function ManifoldsBase.isapprox(
     return false
 end
 
+"""
+    lie_group_dimension(G::LieGroup)
+
+The dimension ``n=$(_tex(:rm, "dim"))_{$(_tex(:Cal,"G"))}`` of real space ``ℝ^n`` to which the neighborhood of
+each point of the [`LieGroup`](@ref) `M` is homeomorphic.
+
+This function is implemented to call [`manifold_dimension`](@extref `ManifoldsBase.manifold_dimension-Tuple{AbstractManifold}`) on the [`AbstractManifold`](@ref) `M`
+within the Lie group `G`.
+"""
+lie_group_dimension(G::LieGroup) = manifold_dimension(G.manifold)
```

Comment:
Since Lie groups are still manifolds, I think isn't particularly useful?

Reply:
we could also just “pass down” then like 

```suggestion
manifold_dimension(G::LieGroup) = manifold_dimension(G.manifold)
```

? I would just also remove the doc string then.

Reply:
Sure, that resolves this issue.